paket.dependencies

```diff
@@ -1,8 +1,9 @@
 source https://nuget.org/api/v2
 
 nuget FSharp.Formatting
-nuget NUnit 2.6.3
-nuget NUnit.Runners 2.6.3
+nuget Newtonsoft.Json
```

Comment:
As I understand it, ```FSharp.Data``` is very useful when working with ```JsonValue``` (records). However, I need to serialize and derserialize the types provided by the SwaggerProvider (Not records). I don't see any easy way to serialize/deserialize these using FSharp.Data. That's why I used ```NewtonSoft.Json``` instead.